### PR TITLE
fix(component): lit-react disappear when modal in close animation

### PR DESCRIPTION
**Pull request by EYHN**

### CODE REVIEW

**Comment by EYHN** [comment]
* **#7624** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7624?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @EYHN and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/component/src/lit-react/create-component.ts [hunk 1]
```diff
@@ -314,13 +314,10 @@ export const createComponent = <
 
     React.useLayoutEffect(() => {
       const container = containerRef.current;
-      if (!container) {
+      if (!container || container.contains(element)) {
```
Comment (pengx17) [inline packages/frontend/component/src/lit-react/create-component.ts:317]: The wrapped lit component will never be unmounted?

Comment (CatsJuice) [reply packages/frontend/component/src/lit-react/create-component.ts:317]: I'm also quite worried, it feels dangerous

Comment (EYHN) [reply packages/frontend/component/src/lit-react/create-component.ts:317]: React will unmount the parent element.

**Pull request closed without merging**